app/views/teams/_conference_edit.html.slim

```diff
@@ -0,0 +1,28 @@
+.page-header
```

Comment:
👍  Can you please rename the partial to `_conference_preference_fields`? It's otherwise confused with the teams form or edit view.